source/javascripts/refills/example.js

```diff
@@ -0,0 +1,8 @@
+$(function() {
+  $("button").click(function() {
```

Comment:
Thanks. I completely agree.

I'd started working on the first component, but half-way through decided to use this a temporary placeholder instead. I plan on discarding all of this.